custom_components/aromalink_ha_integration/sensor.py

```
@@ -191,9 +191,13 @@ def native_value(self):
             "startCount",
             "start_count",
         )
+        if raw is None:
+            return None
+        # runCount accumulates seconds of work time; convert to hours.
+        return round(raw / 3600, 2)
 
 class AromaLinkPumpCountSensor(AromaLinkSensorBase):
-    """Sensor showing the number of times the pump has operated (diffusions)."""
+    """Sensor showing total diffusion time in hours (airPumpCount × work_duration)."""
```

Ruff (0.15.15)

[warning] Docstring contains ambiguous `×` (MULTIPLICATION SIGN). Did you mean `x` (LATIN SMALL LETTER X)?

(RUF002)